src/main/java/com/google/cloud/tools/maven/StageMojo.java

```diff
@@ -268,7 +269,7 @@ protected void configureDockerfileDefaultLocation() {
    * <p>Called during {@link #execute()}, subclasses can override to provide a different value for
    * {@link #appEngineDirectory}.
    */
-  protected void configureAppEngineDirectory() {
+  protected void configureAppEngineDirectory() throws MojoExecutionException {
```

Comment:
Is this `MojoExecutionException` used?

Reply:
This method is overridden in AbstractSingleYamlDeployMojo and it's used there.

Reply:
Hmm, it seems that `isStandardStaging` should probably not set the `appengineWebXml` (like a method that should be a boolean getter should probably not be changing the state of the object).

Reply:
Referring to `AbstractSingleYamlDeployMojo` using `isStandardStaging`, which is why it seems to throw the `MojoExecutionException`.